app/controllers/events_controller.rb

```diff
@@ -1,5 +1,3 @@
-require_relative '../models/event_handler.rb'
```

Comment:
@calebcauthon In general, we want to avoid requiring anything in a Rails application.  It's ok when we're just getting started, but if make sure that our file name is the same as class we're declaring, then Rails will handle the requiring for us.